src/components/molecules/product-card/components/UnavailabilityMessage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+export const UnavailabilityMessage = () => {
```

Comment:
is a bit misaligned 👀 
![image](https://user-images.githubusercontent.com/5409077/58577744-0b8f4500-823f-11e9-87e3-8311e9957a16.png)

Reply:
🤔 it doesn't look like that for me? Guessing it's to do with the font, are you using windows?

Reply:
damn windows
https://i.giphy.com/media/1Syi8XSAkiFupEty1u/giphy.webp